src/video/SDL_egl_c.h

```diff
@@ -115,6 +115,10 @@ typedef struct SDL_EGL_VideoData
 
     EGLint(EGLAPIENTRY *eglClientWaitSyncKHR)(EGLDisplay dpy, EGLSyncKHR sync, EGLint flags, EGLTimeKHR timeout);
 
+    EGLImageKHR(EGLAPIENTRY *eglCreateImageKHR)(EGLDisplay dpy, EGLContext ctx, EGLenum target, EGLClientBuffer buffer, const EGLint *attrib_list);
+
+    EGLBoolean(EGLAPIENTRY *eglDestroyImageKHR)(EGLDisplay dpy, EGLImageKHR image);
```

Comment:
These are officially standardized in EGL 1.5 as `eglCreateImage()` and `eglDestroyImage()`. We should use the standard functions if the EGL version is 1.5 or later.

Reply:
Hi cgutman,
    Do you mean that I can use eglCreateImage() instead of eglCreateImageKHR() after EGL version 1.5?
but I checked the  EGL Specification (https://www.khronos.org/registry/EGL/sdk/docs/man/), the target of eglCreateImage() doesn't include EGL_LINUX_DMA_BUF_EXT.
and the two functions have different types of parameter attrib_list. 
Is it OK that I use eglCreateImageKHR() directly?

Reply:
`EGL_LINUX_DMA_BUF_EXT` is part of `EGL_EXT_image_dma_buf_import` not `KHR_image_base`. You can use it with both `eglCreateImage()` and `eglCreateImageKHR()`.

Reply:
Thank you for the detailed explanation